package.json

```diff
@@ -127,5 +127,8 @@
     "unzipper": "^0.10.14",
     "url-join": "^5.0.0",
     "vitest": "^0.34.1"
+  },
+  "dependencies": {
+    "ky": "^1.1.0"
```

Comment:
this changed should be reverted, we don't use `ky` in the project aside in the `webdriver` package where it is defined